src/index.ts

```diff
@@ -0,0 +1,13 @@
+export { default as camelize } from './helpers/camelize.ts';
+export { default as capitalize } from './helpers/capitalize.ts';
+export { default as classify } from './helpers/classify.ts';
+export { default as dasherize } from './helpers/dasherize.ts';
+export { default as htmlSafe } from './helpers/html-safe.ts';
+export { default as humanize } from './helpers/humanize.ts';
+export { default as lowercase } from './helpers/lowercase.ts';
+export { default as titleize } from './helpers/titleize.ts';
+export { default as trim } from './helpers/trim.ts';
+export { default as truncate } from './helpers/truncate.ts';
+export { default as underscore } from './helpers/underscore.ts';
+export { default as uppercase } from './helpers/uppercase.ts';
+export { default as w } from './helpers/w.ts';
```

Comment:
is this wise? sure we can give people the convenience of importing from a barrel file but should we not just be encouraging them import the helpers directly? 

And since we only export helpers we could add an exports rule that means that they just import from `ember-cli-string-helpers/lowercase` rather than always needing the `/helpers/` in the path. 

This is only a mild suggestion, feel free to ignore 👍